docs/source/quickstart.rst

```diff
@@ -27,18 +27,16 @@ especially when setting up the ``DEFAULT_PERMISSIONS`` (which apply to not-logge
 
 Here is the documentation straight from its config:
 
-.. autoclass:: bepasty.config.Config
+.. automodule:: bepasty.config
```

Comment:
@ThomasWaldmann Could you have a look at the generated docs and share your opinion on the rendering from `.. automodule::` ? I find that it doesn't look too well integrated into the documentation. I liked the automatic `[source]` link and header stylistic provided by `.. autoclass::` but I don't think this is available for `.. automodule::`. 

To make it look a *little* bit better, there's a setting `add_module_names = True` in `conf.py` which removes the prefix from the attributes. It helps the aesthetics a little bit, and we could also move the disclaimer from the docstring to elsewhere.

```
IMPORTANT:

The config is only loaded at startup time of the app, so if you change it,
you need to restart the wsgi app process(es) to make it load the updated
config.
```

Reply:
True is the default, but guess it looks better with add_module_names = False - so please add that below the comment describing this in conf.py.

Reply:
OK finally, I don't think the attribute prefixes look so bad if the modules' own docstring is moved into the documentation.

I modified the documentation very slightly to accommodate a "key:value" type source code, and I think it looks pretty good.

What do you think of ffc0c205bcd588d2d4965f2b81140419c4459db4 ?

Reply:
I commented there...